lib/matplotlib/_constrained_layout.py

```diff
@@ -266,10 +267,14 @@ def _make_layout_margins(ax, renderer, h_pad, w_pad):
     """
     fig = ax.figure
     invTransFig = fig.transFigure.inverted().transform_bbox
-
     pos = ax.get_position(original=True)
     tightbbox = ax.get_tightbbox(renderer=renderer)
     bbox = invTransFig(tightbbox)
+    # this can go wrong:
+    if not np.isfinite(bbox.y0 + bbox.x0 + bbox.y1 + bbox.x1):
```

Comment:
same comment as elsewhere: `isfinite(bbox.width) and isfinite(bbox.height)` (just for clarity... IMO?)